sc2reader/events/game.py

```diff
@@ -685,8 +689,11 @@ def __init__(self, frame, pid, data):
         self.custom_resource = self.resources[3] if len(self.resources) >= 4 else None
 
     def __str__(self):
-        return self._str_prefix() + " requests {0} minerals, {1} gas, {2} terrazine, and {3} custom".format(
-            self.minerals, self.vespene, self.terrazine, self.custom_resource
+        return (
+            self._str_prefix()
+            + " requests {0} minerals, {1} gas, {2} terrazine, and {3} custom".format(
```

Comment:
The {0} {1} {2} {3} thing has not been needed since Python 2.5 unless you want substitutions to land out of order or you want substitutions to repeat.

Reply:
This is how it was originally written and how the other classes in this file are written as well. Maybe create a separate issue to change this more broadly?

Reply:
Will automate it with https://github.com/asottile/pyupgrade

Reply:
Sounds good to me